modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -184,4 +196,229 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
       }
     }
   }
+
+  "authenticate for V2" should {
+    lazy val route: Route = authenticate(executor, authenticatorV2, timeout, requestValidationTimeout).apply(complete(HttpResponse()))
+    val publicKey = MAuthKeysHelper.getPublicKeyFromString(FixturesLoader.getPublicKey)
+
+    "pass successfully authenticated request with parameters" in {
+      (client.getPublicKey _).expects(appUuid).returns(Future(Some(publicKey)))
+      //noinspection ConvertibleToMethodValue
+      (mockEpochTimeProvider.inSeconds _: () => Long).expects().returns(timeHeader)
+
+      Get("/").withHeaders(
+        RawHeader(MAuthRequest.MCC_TIME_HEADER_NAME, timeHeader.toString),
+        RawHeader(MAuthRequest.MCC_AUTHENTICATION_HEADER_NAME, authHeaderV2),
+        RawHeader(MAuthRequest.X_MWS_TIME_HEADER_NAME, timeHeaderV2.toString),
+        RawHeader(MAuthRequest.X_MWS_AUTHENTICATION_HEADER_NAME, authHeader)
+      ) ~> route ~> check {
+        status shouldEqual StatusCodes.OK
+      }
+    }
+
+    "pass successfully authenticated request with V2 headers" in {
+      (client.getPublicKey _).expects(appUuid).returns(Future(Some(publicKey)))
+      //noinspection ConvertibleToMethodValue
+      (mockEpochTimeProvider.inSeconds _: () => Long).expects().returns(timeHeader)
+
+      Get("/").withHeaders(
+        RawHeader(MAuthRequest.MCC_TIME_HEADER_NAME, timeHeader.toString),
+        RawHeader(MAuthRequest.MCC_AUTHENTICATION_HEADER_NAME, authHeaderV2)
+      ) ~> route ~> check {
+        status shouldEqual StatusCodes.OK
+      }
+    }
+
+    "reject if request validation timeout passed" in {
+      (client.getPublicKey _).expects(*).never
+      //noinspection ConvertibleToMethodValue
+      (mockEpochTimeProvider.inSeconds _: () => Long).expects().returns(timeHeader)
+
+      Get().withHeaders(
+        RawHeader(MAuthRequest.MCC_TIME_HEADER_NAME, (timeHeader - requestValidationTimeout.toSeconds - 10).toString),
+        RawHeader(MAuthRequest.MCC_AUTHENTICATION_HEADER_NAME, authHeaderV2),
+        RawHeader(MAuthRequest.X_MWS_TIME_HEADER_NAME, (timeHeader - requestValidationTimeout.toSeconds - 10).toString),
+        RawHeader(MAuthRequest.X_MWS_AUTHENTICATION_HEADER_NAME, authHeader)
+      ) ~> route ~> check {
+        inside(rejection) { case MdsolAuthFailedRejection ⇒ }
```

Comment:
Can you update these unicode character to be `=>`?

Reply:
fixed https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/735bd0daeb6afd8ec05d1562413a7721f033797f